src/__tests__/commands/flex/plugins/start.test.ts

```diff
@@ -8,16 +8,16 @@ describe('Commands/FlexPluginsStart', () => {
     sinon.restore();
   });
 
-  start()
-    .setup((instance) => {
-      sinon.stub(instance, 'runScript').returnsThis();
+  start(['start', '--name', 'plugin-testOne', '--name', 'plugin-testTwo', '--include-remote'])
```

Comment:
Please add a few more tests, you want to test the functionality with no `--name`, one `--name` and multiple `--name`  and one with no `--include-remote` and one with `--include-remote`